src/Application/Kernel.php

```diff
@@ -8,39 +8,38 @@
 use RecursiveIteratorIterator;
 use ReflectionClass;
 use Tempest\AppConfig;
-use Tempest\Application\Exceptions\KernelException;
 use Tempest\Container\Container;
 use Tempest\Container\GenericContainer;
 use Tempest\Database\PDOInitializer;
-use Tempest\Discovery\DiscoveryLocation;
 use Tempest\Http\RequestInitializer;
 use Tempest\Http\RouteBindingInitializer;
 use Tempest\Http\ServerInitializer;
-use function Tempest\path;
 use Throwable;
 
 final readonly class Kernel
 {
     public function __construct(
-        private string $root,
+        public string $root,
         private AppConfig $appConfig,
+        private array $bootstraps = [],
     ) {
     }
 
     public function init(): Container
     {
-        $container = $this->initContainer();
-
-        $this->initDiscoveryLocations();
+        $container = $this->createContainer();
+        $container->config($this->appConfig);
 
-        $this->initConfig($container);
+        foreach ($this->bootstraps as $bootstrap) {
+            $container->get($bootstrap)->boot();
+        }
 
         $this->initDiscovery($container);
 
         return $container;
     }
 
-    private function initContainer(): Container
+    private function createContainer(): Container
```

Comment:
What's the reason for renaming this?

Reply:
Part of @brendt's recommendations in the issue.